addons/mail/static/src/core/common/message_model.js

```diff
@@ -371,6 +367,8 @@ export class Message extends Record {
         }
     }
 
+    _onUpdateIsEmpty() {}
```

Comment:
Can't we wait for the fw of the fix PR and drop this?
Or even just drop it preemptively if we want to merge this first?

Reply:
dropped preemptively